mobileportfolio.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Mobile Portfolio</title>
+    <link rel="stylesheet" href="mobileportfolio.css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <main class="bg-shapes">
+       <header class="logo-nav">
+            <nav class="navbar">
+                <div class="my-logo">My Logo</div>
+                <a href="#" class="toggle-button">
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                    <span class="bar"></span>
+                </a>
```

Comment:
Kindly include the hamburger menu at the top right of the page. This will be used to toggle the display of the mobile menu.

Your design:
![image](https://user-images.githubusercontent.com/20554583/219360417-3360b85d-71fd-4d2b-a406-16a463162be6.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/219360460-814a9121-23f9-4a3e-9b7d-52790235bbd0.png)

Reply:
@emmiiorji Kindly note that there is a toggle in the project. What can be the reason for the toggle not showing and how do I go about it?